[2024-10-01T13:17:04.499+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T13:17:04.521+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_data.success manual__2024-10-01T13:16:56.022162+00:00 [queued]>
[2024-10-01T13:17:04.542+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_data.success manual__2024-10-01T13:16:56.022162+00:00 [queued]>
[2024-10-01T13:17:04.543+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T13:17:04.567+0000] {taskinstance.py:2888} INFO - Executing <Task(_BashDecoratedOperator): success> on 2024-10-01 13:16:56.022162+00:00
[2024-10-01T13:17:04.576+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1014) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-01T13:17:04.580+0000] {standard_task_runner.py:72} INFO - Started process 1016 to run task
[2024-10-01T13:17:04.581+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transform_data', 'success', 'manual__2024-10-01T13:16:56.022162+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/transform_data.py', '--cfg-path', '/tmp/tmp8nihf48e']
[2024-10-01T13:17:04.586+0000] {standard_task_runner.py:105} INFO - Job 196: Subtask success
[2024-10-01T13:17:04.702+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_data.success manual__2024-10-01T13:16:56.022162+00:00 [running]> on host 9ff0d37a02a4
[2024-10-01T13:17:04.909+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transform_data' AIRFLOW_CTX_TASK_ID='success' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T13:16:56.022162+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T13:16:56.022162+00:00'
[2024-10-01T13:17:04.924+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T13:17:04.987+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-01T13:17:04.998+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "The DAG is running well"']
[2024-10-01T13:17:05.033+0000] {subprocess.py:86} INFO - Output:
[2024-10-01T13:17:05.037+0000] {subprocess.py:93} INFO - The DAG is running well
[2024-10-01T13:17:05.039+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-01T13:17:05.073+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T13:17:05.074+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=transform_data, task_id=success, run_id=manual__2024-10-01T13:16:56.022162+00:00, execution_date=20241001T131656, start_date=20241001T131704, end_date=20241001T131705
[2024-10-01T13:17:05.121+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-01T13:17:05.145+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-01T13:17:05.153+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
